### PR TITLE
Firt review iteration - First changes in Title, Introduction and other Rules

### PR DESCRIPTION
**Pull request by ypriverol**

### CODE REVIEW

File: document/manuscript.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Ten Simple Rules for taking advantage of GitHub in bioinformatics
+#Ten Simple Rules for taking advantage of GitHub
```
Comment (ypriverol) [inline document/manuscript.md:1]: This change will solve the issue #52

File: document/manuscript.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-Yasset Perez-Riverol (1)[^1], Rui Wang (1), Timo Sachsenberg (2), Julian Uszkoreit (3), Laurent Gatto (4), Felipe da Veiga Leprevost (5), Christian Fufezan (6), Juan Antonio Vizcaíno (1)[^2]
+Yasset Perez-Riverol (1)[^1], Rui Wang (1), Timo Sachsenberg (2), Julian Uszkoreit (3), Laurent Gatto (4), Felipe da Veiga Leprevost (5), Christian Fufezan (6), Tobias Ternent (1), Juan Antonio Vizcaíno (1)[^2]
```
Comment (ypriverol) [inline document/manuscript.md:3]: @tobias-ternent has started to contributed with the manuscript

File: document/manuscript.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ projects, since they enable the organisation and sharing of
 programming tasks between different remote contributors. Here, we
 introduce the main features of GitHub, a popular web-based platform
 which offers a free and integrated environment for hosting the source
-code, documentation and web page for open source projects. GitHub also
+code, documentation and project-related web content for open source projects. GitHub also
```
Comment (ypriverol) [inline document/manuscript.md:44]: Refinements related with issue #53  and **Reviewer 3** comments

File: document/manuscript.md [hunk 4]
```diff
@@ -49,27 +49,37 @@ Linux kernel. One reason for GitHub's success is that it offers more
 than a simple source code hosting service. It provides developers with
 a dynamic and collaborative environment, often coined as social coding
 platform, with the ability to review, comment and discuss code
-\cite{Dabbish:2012}.
+\cite{Dabbish:2012}. Individual bioinformatics projects (https://github.com/lgatto/MSnbase) \cite{Gatto15012012},
```
Comment (ypriverol) [inline document/manuscript.md:52]: refinements in issue #53 and comments from **Reviewer 2**

File: document/manuscript.md [hunk 5]
```diff
@@ -49,27 +49,37 @@ Linux kernel. One reason for GitHub's success is that it offers more
...
 
 Some of our recommendations outlined below are applicable to other
 hosting services. However our main aim here is to highlight specific
 GitHub features. We provide a set of recommendations to take full
 advantage of GitHub's features to manage small and large
 bioinformatics projects and increase their profile and
-visibility. These rules have been ordered to reflect a typical
-development process: learning git and GitHub basics, use branches,
-label and tag code, track the project using issues, etc.
+visibility. These rules have been ordered to reflect a typical development process:
```
Comment (ypriverol) [inline document/manuscript.md:62]: Refinements in issue #53 comments from **reviewer 3**

File: document/manuscript.md [hunk 6]
```diff
@@ -49,27 +49,37 @@ Linux kernel. One reason for GitHub's success is that it offers more
...
 bioinformatics projects and increase their profile and
-visibility. These rules have been ordered to reflect a typical
-development process: learning git and GitHub basics, use branches,
-label and tag code, track the project using issues, etc.
+visibility. These rules have been ordered to reflect a typical development process: 
+learning git and GitHub basics, use of branches, labeling and tagging of code snapshots,
+tracking project bugs and enhancements using issues, and disemination of the final results.
 
-## Rule 1. Structure your projects: users, organisations, repositories and teams
+## Rule 1. Structure your projects: repositories, users, organisations and teams
```
Comment (ypriverol) [inline document/manuscript.md:66]: The order of concepts has been changed as requested by reviewer 2 in issue #59

File: document/manuscript.md [hunk 7]
```diff
@@ -49,27 +49,37 @@ Linux kernel. One reason for GitHub's success is that it offers more
...
 
 Open source projects on GitHub are visible to everyone, but write
 permissions, i.e. the possibility to directly modify the content, need
-to explicitly be granted. On the other hand, everyone with a GitHub
-account can fork any public project and start developing in one's own
-fork. This forking is the basis of social coding. It allows anyone to
-develop and test novel features into existing code and offers the
+to explicitly be granted. One of the key concepts in GitHub is the repositories where
```
Comment (ypriverol) [inline document/manuscript.md:70]: The order of concepts has been changed as requested by reviewer 2 in issue #59

File: document/manuscript.md [hunk 8]
```diff
@@ -49,27 +49,37 @@ Linux kernel. One reason for GitHub's success is that it offers more
...
+there is nothing preventing users from keeping text documents such as analysis reports and manuscripts (see for example the repository for this
+manuscript at https://github.com/ypriverol/github-paper), or other file types in your projects. Note that until recently, GitHub was lacking
+support for storing large files (>100 MB), a issue that has been recently addressed by the GitHub large file storage. 
+Everyone with a GitHub account can fork any public repository and start developing in one's own
+fork. A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes
+without affecting the original project (https://help.github.com/articles/fork-a-repo/). This forking is
+the basis of social coding. It allows anyone to develop and test novel features into existing code and offers the
 possibility to merge novel features back the into the main project,
-thereby becoming a contributor.  Structuring your projects allows to
+thereby becoming a contributor.  Project managers can structure projects to
```
Comment (ypriverol) [inline document/manuscript.md:82]: Change included as suggested by **reviewer 2** in issue #59

File: document/manuscript.md [hunk 9]
```diff
@@ -285,7 +284,8 @@ as it needs to have an appropriate license
 
 ## Rule 9. Promote your projects in the scientific community - create a web page and more
 
-Additional steps can be done to boost the visibility of a
+The traditional and effective way of promoting scientific software is by
```
Comment (ypriverol) [inline document/manuscript.md:287]: Issue #57 about the promotion using per-reviewed journals

File: document/manuscript.md [hunk 10]
```diff
@@ -213,7 +212,7 @@ and badges (also called shields) for the projects that can be included
 on your GitHub page, thereby making your projects easily identifiable
 as high quality and well-maintained.
 
-## Rule 7. Use and maintain your GitHub issue trackers
+## Rule 7. Use and maintain your issue trackers
```
Comment (ypriverol) [inline document/manuscript.md:215]: taking into account the suggestion of issue #58 we will make some of the titles more generic

File: document/manuscript.md [hunk 11]
```diff
@@ -250,14 +249,15 @@ produce DOIs for other types of scientific outputs such as datasets
 \cite{Vizcaino:2014} or training materials (for example
 \cite{Ahmadia_2015_27353}). The main motivation behind this is to give
 scientists a better credit for their work \cite{NatBiotechEditorial:2009},
-enabling at the same time a better way to cite and track it. A common
-issue with software is that it normally evolves at a different speed
+enabling at the same time a better way to cite and track it. In addition, funding agencies such as NIH now accept
```
Comment (ypriverol) [inline document/manuscript.md:252]: changes related with issue #60 **reviewer 3**

File: document/manuscript.md [hunk 12]
```diff
@@ -250,14 +249,15 @@ produce DOIs for other types of scientific outputs such as datasets
...
+enabling at the same time a better way to cite and track it. In addition, funding agencies such as NIH now accept 
+other research products as citable research such as accessible datasets and softwares.
+A common issue with software is that it normally evolves at a different speed
 than what is published in the scientific literature. In fact, it is
 common to find software having novel features and functionalities that
 were not described in the original publication. GitHub now enables the
 use of DOIs to cite the code deposited, using the data archiving tool
 Zenodo (https://zenodo.org/). The procedure is simple (see
-https://guides.GitHub.com/activities/citable-code/) and, by default,
+https://guides.github.com/activities/citable-code/) and, by default,
```
Comment (ypriverol) [inline document/manuscript.md:260]: changes related with issue #60 reviewer 3

**Comment by ypriverol** [comment]
@all Can anyone accept this pull-request

**Comment by ypriverol** [comment]
@lgatto @timosachsenberg @Leprevost can you review the current pull-request?

File: document/manuscript.md (hunk 3 above)
Comment (timosachsenberg) [inline document/manuscript.md:44]: why not only: "web content"
project-related is a bit redundant given the sentence ends with projects

Comment (ypriverol) [inline document/manuscript.md:44]: @timosachsenberg I was trying to fit the reviewers comments, then I add this, but if you feel is redundant please remove it

File: document/manuscript.md (hunk 7 above)
Comment (timosachsenberg) [inline document/manuscript.md:70]: "are repositories, the place where..."

File: document/manuscript.md [hunk 13]
```diff
@@ -285,7 +284,8 @@ as it needs to have an appropriate license
 
 ## Rule 9. Promote your projects in the scientific community - create a web page and more
 
-Additional steps can be done to boost the visibility of a
+The traditional and effective way of promoting scientific software is by
+publishing them in the peer-reviewed scientific literature. Additional steps can be done to boost the visibility of a
```
Comment (timosachsenberg) [inline document/manuscript.md:288]: an organization

**Comment by prvst** [comment]
+1

**Comment by ypriverol** [comment]
@lgatto @timosachsenberg can anyone merge this pull-request?

**Comment by timosachsenberg** [comment]
its out of sync

**Comment by ypriverol** [comment]
Ok let me try to sync and create another pull-request

**Pull request closed without merging**